src/amo/sagas/reviews.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+/* global Generator */
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+
+import { getReviews } from 'amo/api';
+import { setAddonReviews } from 'amo/actions/reviews';
+import { FETCH_REVIEWS } from 'amo/constants';
+import log from 'core/logger';
+import { createErrorHandler, getApi } from 'core/sagas/utils';
+import type { FetchReviewsAction } from 'amo/actions/reviews';
+
+
+export function* fetchReviews(
+  {
+    payload: { errorHandlerId, addonSlug, page },
+  }: FetchReviewsAction
+): Generator<any, any, any> {
+  const errorHandler = createErrorHandler(errorHandlerId);
+  try {
+    yield put(showLoading());
+    const api = yield select(getApi);
+    const response = yield call(getReviews, {
+      // Hide star ratings (reviews that do not have a body).
```

Comment:
This could be clarified slightly: "Hide star-only ratings (reviews that do not have a body)".

All ratings have stars, so "star-only" might help the reader grok this a bit faster.